source/release-notes/1.7.txt

```diff
@@ -31,6 +31,27 @@ Synchronize Data Between Clusters with Different MongoDB Server Major Versions
 
 .. include:: /includes/cross-version-sync.rst
 
+Other Notes
+~~~~~~~~~~~
+
+New Features:
+
+- ``mongosync`` now truncates logged fields that contain more than 1,024
+  characters.
+
+Issues Fixed:
+
+- Fixed bug introduced in ``mongosync`` 1.5.0 that may cause ORR to insert
+  capped collection documents out of order. That is, documents on the source
+  and destination cluster may not be deleted from the capped collection in the
+  same order. This bug is only triggered if documents are inserted into a
+  capped collection during migration, before or while the Collection Copy phase
+  is working on the same capped collection.
+
+- Fixed bug introduced in 1.6 that caused ``mongosync`` to fail during 
+  initialization if the user specified only an exclusion filter and there were
+  no fully excluded databases.
+
 Minimum Supported Version
```

Comment:
I know this PR did not change this section. But the minimum supported versions should be 6.0.8 and 7.0.0 (unchanged from 1.6). Is it possible to correct this line in this PR?

Reply:
Sure

Reply:
Fixed